src/templates/use-cases.tsx

```diff
@@ -324,22 +324,47 @@ const UseCasePage = ({
       {
         text: t("template-usecase-dropdown-defi"),
         to: "/defi/",
+        matomo: {
+          eventCategory: "use cases menu",
+          eventAction: "click",
+          eventName: "/defi/",
```

Comment:
Just noting that this differs from the requested eventName in the Notion card. Those don't have the `/`s. Maybe is not a problem and we can leave it like is.

```suggestion
          eventName: "defi",
```

Reply:
Same with the other cases in this file.

Reply:
Pushed up changes for this